source/tyr/tyr/resources.py

```diff
@@ -684,6 +684,35 @@ def put(self, version=0, id=None, name=None):
                 default=getattr(instance, "max_{}_direct_path_duration".format(mode)),
             )
 
+        parser.add_argument(
```

Comment:
why do not add `max_car_no_park_direct_path_duration` like the others ?

Reply:
Because it is already contained in https://github.com/CanalTP/navitia/blob/7ba713cab17a69ec6b2b0c9076f0ea17d62d5b94/source/tyr/tyr/resources.py#L679